### PR TITLE
Add images to deposit/withdraw references

### PR DESCRIPTION
**Pull request by vitchor**

### CODE REVIEW

File: content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ These docs walk through the steps necessary to set up and launch both a test ser
 
 To help with server setup, SDF created [Polaris](https://github.com/stellar/django-polaris), an extendable django reusable-app that comes with fully-implemented endpoints, templates, and database models. It's built in Python using the community-supported [Stellar Python SDK](https://github.com/StellarCN/py-stellar-base), and is compliant with the Stellar Ecosystem Protocols mentioned in the previous section.
  
-**Image 1: Screenshots of the reference implementation**
+![Screenshots of the reference implementation](web-assets/polaris.png)
```
Comment (vcarl) [inline content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx:12]:
```suggestion
![Screenshots of the reference implementation](../web-assets/polaris.png)
```

File: content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx [hunk 2]
```diff
@@ -21,14 +21,14 @@ Once you have a server set up, SDF also maintains a [Demo Client Project](https:
 
 We've even deployed a [Deployed Demo Client](https://sep24.stellar.org/), which allows you to run those tests with a UI without needing to set up a new hosting infrastructure for that project.
 
-**Image 2: Screenshots of the demo client**
+![Screenshot of the demo client](web-assets/demo-client.png)
```
Comment (vcarl) [inline content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx:24]:
```suggestion
![Screenshot of the demo client](../web-assets/demo-client.png)
```

File: content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx [hunk 3]
```diff
@@ -21,14 +21,14 @@ Once you have a server set up, SDF also maintains a [Demo Client Project](https:
...
 
 ## Anchor Validation Suite
 The SEP-24 Anchor Validation Suite is a set of tests that helps confirming if your anchor implementation is compliant with the latest specifications of the SEP-24 standard. This is a great resource for people that are getting started building an anchor and want to check what's still missing in their implementation, or for teams that have already finalized the anchor development and want an extra validation on the codebase. Also, the Anchor Validation Suite doesn't replace the necessity of having a professional security auditor reviewing the whole project.
 
-**Image 3: of the Anchor Validation Suite's UI**
+![Screenshot of the Anchor Validation Suite's UI](web-assets/anchor-validation-suite.png)
```
Comment (vcarl) [inline content/docs/enabling-deposit-and-withdrawal/reference-implementations.mdx:31]:
```suggestion
![Screenshot of the Anchor Validation Suite's UI](../web-assets/anchor-validation-suite.png)
```

File: content/docs/enabling-deposit-and-withdrawal/setting-up-test-server.mdx [hunk 4]
```diff
@@ -177,6 +177,8 @@ Wallets poll the `/transaction` endpoint when waiting for a transaction’s stat
 
 Wallets need access to the transactions associated with the user’s account for displaying history or past activity. This can be accomplished with a `/transactions` endpoint. Wallets may also use this endpoint after making a POST request to `/transactions/deposit/interactive` to make sure a transaction was successfully created.
 
+![Status Diagram of SEP-24 transactions](web-assets/SEP24-state-diagram.png)
```
Comment (vcarl) [inline content/docs/enabling-deposit-and-withdrawal/setting-up-test-server.mdx:180]:
```suggestion
![Status Diagram of SEP-24 transactions](../web-assets/SEP24-state-diagram.png)
```

**Comment by vitchor** [comment]
Thanks @vcarl !

**Comment by jeesunikim** [comment]
Closing this and created a new [PR #154](https://github.com/stellar/new-docs/pull/154) to resolve conflicts so we can merge.

**Pull request closed without merging**